### DISCUSSION TITLE
Camera hung errors "mainly" in daytime mode

### DISCUSSION THREAD

**Discussion by czimm74** [aaronwmorris/indi-allsky#961] [General]
Ok, with my rebuild I am running a mini linux PC that is connected to a PlayerOne Neptune-CII camera.  Night time it is capturing fine.  But when I go into daytime, when I log in to the website it tells me that the camera might be hung.

I have tried with daytime capture off, daytime on but no time lapse and daytime with time lapse.  I get a camera being hung up no matter what.  And if it is hung, it won't go into nighttime capture.  

I am wondering, before in testing I didn't notice this error.  So could be be from using too long to a USB extension cable?  I ended up being ~2' shy of a direct connection.  But I would expect this to also affect nighttime captures as well.

**Comment by aaronwmorris** [comment]
The cable length might be the cause.  You might try adjusting/lowering the USB Bandwidth.  I updated the wiki with the USB bandwidth setting that I found for my PoA camera.

https://github.com/aaronwmorris/indi-allsky/wiki/INDI-custom-config

**Reply by czimm74** [reply]
Tried that adjustment, seems like it is working.  Camera did wake up at night mode.  But I also ordered a better usb extension, the one I am using is about 15 years old.  Also shorter length.

**Comment by aaronwmorris** [comment]
There is also a script you can use to kind of torture test your camera.  `./testing/exposureTest.py`

You can populate the `CCD_EXPOSURES` and `CCD_GAIN` lists and it will loop through the exposures and gains.  The exposures are not saved anywhere.

**Reply by czimm74** [reply]
May try that when I get the new cable, then do a comparison between the two.
